### PR TITLE
Single window mode is not lost after starting a project from the project manager 

### PR DESCRIPTION
**Pull request by jak6jak**
This is my first pull request ever so hopefully this goes smoothly. This pull request implements this junior issue: https://github.com/godotengine/godot/issues/42034

Hopefully everything is in order thanks!

### CODE REVIEW

File: editor/project_manager.cpp [hunk 1]
```diff
@@ -2023,6 +2023,9 @@ void ProjectManager::_open_selected_projects() {
 			args.push_back("--disable-crash-handler");
 		}
 
+		if (OS::get_singleton()->is_single_window()) {
+			args.push_back("--single-window");
+		}
```
Comment (pouleyKetchoupp) [inline editor/project_manager.cpp:2028]: Please add an empty line here to separate the block from the next line.

**Comment by pouleyKetchoupp** [comment]
It works fine. Could you please squash your commits together?
https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html?#the-interactive-rebase

**Comment by jak6jak** [comment]
I followed the instructions and it looks like it squashed them locally. I force pushed them but it doesn't look like anything has changed here on GitHub. Do you happen to know what I might have missed?

**Comment by pouleyKetchoupp** [comment]
It looks like the test branch wasn't pushed. Can you try to force push your test branch again?

**Comment by jak6jak** [comment]
I've pushed multiple times and it says that everything is up to date. This is the command I'm using:
git push --force origin test

**Comment by pouleyKetchoupp** [comment]
Maybe this commit has been messing with your branch state somehow:
` Merge branch 'test' of https://github.com/jak6jak/godot into test `

Do you use a UI tool like GitExtensions? A screenshot of your current local/remote branch status could help.

**Comment by jak6jak** [comment]
I got some help and they told me to make a new branch and retry. Here it is: https://github.com/godotengine/godot/pull/42099

**Comment by aaronfranke** [comment]
@jak6jak If you have Git trouble in the future, since you already tried reading the PR workflow article, you might want to try a graphical Git client. I made this video about how to use GitKraken: https://www.youtube.com/watch?v=0XKy8YpP5Ww

**Pull request closed without merging**